media/css/pebbles/base/elements/_document.scss

```diff
@@ -6,12 +6,12 @@
 
 
 html {
-    font-size: 100%;
+    font-size: 18px;
```

Comment:
Yeah I was a little wary of doing this as well, but I think your suggestion sounds sensible. I'll play with it and see how it looks.